core/commands/pubsub.go

```diff
@@ -118,16 +122,33 @@ To use, the daemon must be run with '--enable-pubsub-experiment'.
 	},
 	Marshalers: cmds.MarshalerMap{
 		cmds.Text: getPsMsgMarshaler(func(m *floodsub.Message) (io.Reader, error) {
+			if m.Message == nil {
```

Comment:
No I can't. We use `m.Data` later, which is `m.Message.Data`, which is a nil dereference.